env.example

```diff
@@ -65,6 +65,11 @@ SAML_ENTITY_ID=http://localhost:3000/sp
 # SECRET = cookie session SECRET. If left empty, one will be set automatically
 SECRET=
 
+# ADMINISTRATORS is a list (space delimited) of users who have administrator
+# rights. Use the user's nameID (user2@example.com) or hashed version of
+# nameID ("2b3b2b9ce8").  Either will work.
+ADMINISTRATORS="user1@example.com"
```

Comment:
" are not friendly with docker-compose up --build, @manekenpix help me test this,  and @raygervais  witnessed,
what is weird though if you run the docker-compose up login elasticsearch redis and then npm start telescope it worked with the ", however removing it made it work both ways.